packages/theme/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quid/theme",
-  "version": "1.30.4",
+  "version": "1.30.5",
```

Comment:
CI bumps version, we don't have to do it

Reply:
this has been moved to https://github.com/quid/refraction/pull/50 where package.json is not included.